doc/user/backup_config.md

```diff
@@ -28,12 +28,24 @@ If configurations for both levels are specified then the cluster level configura
 
 ### Operator level configuration  
 
-See the [S3 backup deployment](../../example/deployment-s3-backup.yaml.template) template on how to configure the operator to enable S3 backups. The following flags need to be passed to operator:
+The operator supports AWS credential configuration in several ways.
+* Environment variables
```

Comment:
This is not something we need to explain in this doc. AWS docs should explain this.